lldb/source/Target/Statistics.cpp

```diff
@@ -74,7 +75,7 @@ json::Value ModuleStats::ToJSON() const {
 
   if (!symfile_modules.empty()) {
     json::Array symfile_ids;
-    for (const auto symfile_id: symfile_modules)
+    for (const auto symfile_id : symfile_modules)
```

Comment:
Is this just a whitespace change?

Reply:
Yes I can revert that, probs a formatting on save that I didn't notice.